docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -1,13 +1,15 @@
 # Natural Language Processing (NLP)
 
+> [!NOTE]
+> NLP API is deprecated since Tizen 7.5 and will be removed after two releases without any alternatives.
 
 NLP is a subset of Natural Language Toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (NLP) functionalities, such as language detection, parts of speech, word tokenization, and named entity detection. For more information, see the [NLTK Forum](http://www.nltk.org/).
 
```

Comment:
```suggestion
NLP is a subset of Natural Language Toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (NLP) functionalities, such as language detection, parts of speech, word tokenization, and named entity detection. For more information, see the [NLTK Forum](http://www.nltk.org/).

> [!NOTE]
> NLP API is deprecated since Tizen 7.5 and will be removed after two releases without any alternatives.

```